setup.py

```diff
@@ -453,7 +453,6 @@ def _build_scintilla(self):
             assert os.path.isdir(build_temp), build_temp
         makeargs.append("SUB_DIR_O=%s" % build_temp)
         makeargs.append("SUB_DIR_BIN=%s" % build_temp)
-        makeargs.append("DIR_PYTHON=%s" % sys.prefix)
```

Comment:
If you'd like to restore the functionality of regenerating `scintillacon.py` on every rebuild I can do it, I just need to add a line like
`  powershell -Command "(Get-Content '..\pywin\scintilla\scintillacon.py' | Where-Object { -not $$_.StartsWith('def ') }) | Set-Content '..\pywin\scintilla\scintillacon.py'"
` to the makefile because for some reason under this context h2py finds and adds more definitions, namely broken callables.